tests/impl.h

```diff
@@ -121,6 +121,7 @@
     TYPE(mm_stream_ps)                \
     TYPE(mm_sub_ps)                   \
     TYPE(mm_sub_ss)                   \
+    TYPE(mm_sub_pd)                   \
```

Comment:
The intrinsic type is `SSE2`.
Please place it in the correct classification.

Reply:
Reference: https://software.intel.com/sites/landingpage/IntrinsicsGuide/#techs=SSE2